docs/Manpage.md

```diff
@@ -3545,8 +3545,12 @@ User-specific environment files take precedence over prefix-specific files and
 prefix-specific files take precedence over system-wide files (unless
 `HOMEBREW_SYSTEM_ENV_TAKES_PRIORITY` is set, see below).
 
-Note that these files do not support shell variable expansion e.g. `$HOME` or
-command execution e.g. `$(cat file)`.
+>[!IMPORTANT]
+>
+>Note that Homebrew's environment files are <ins>not</ins> shell scripts.
+>
+>- They do **not** support shell variable expansion, e.g. `$HOME` or command execution, e.g. `$(cat file)`.
+>- They **cannot** parse the "`export`" command syntax, and should <ins><em>only</em></ins> contain key-value pairs, e.g. `HOMEBREW_ENV_VAR=VALUE`
```

Comment:
This needs to be in `brew.1.md.erb` instead and should not contain HTML or GitHub-specific Markdown. Most of the formatting here can be trimmed to better fit the existing style.